src/v/kafka/server/handlers/alter_configs.cc

```diff
@@ -371,14 +371,25 @@ ss::future<response_ptr> alter_configs_handler::handle(
         auto responses = make_audit_failure_response<
           alter_configs_resource_response,
           alter_configs_resource>(
-          std::move(groupped), std::move(unauthorized_responsens));
+          std::move(groupped),
+          std::vector<alter_configs_resource_response>{
+            std::move_iterator(unauthorized_responsens.begin()),
+            std::move_iterator(unauthorized_responsens.end())});
+
+        std::vector<chunked_vector<alter_configs_resource_response>>
+          responses_cv;
+        responses_cv.reserve(responses.size());
+        for (const auto& v : responses) {
+            responses_cv.emplace_back(
+              std::move_iterator(v.begin()), std::move_iterator(v.end()));
+        }
```

Comment:
maybe: `std::move(v.begin(), v.end(), std::back_inserter(responses_cv))` ?

Reply:
ya, this is sort of field surgery at this point. just trying to get something building for scale tests 😇